docs/zh_CN/Compression/CompressionReference.rst

```diff
@@ -140,9 +140,6 @@ Quantizers
 ..  autoclass:: nni.compression.pytorch.utils.shape_dependency.GroupDependency
     :members:
 
-..  autoclass:: nni.compression.pytorch.utils.mask_conflict.CatMaskPadding
-    :members:
-
```

Comment:
I don't know if we can directly modify the CN doc in this place, please check this. And others look great, I will approve this pr.

Reply:
We cannot directly modify the CN docs, have reverted the modification.